Plugin/CppApi/source/CoordinateConversionController.cpp

```diff
@@ -290,6 +290,9 @@ void CoordinateConversionController::setInputGarsConversionMode(CoordinateConver
 {
   m_inputGarsConversionMode = inputGarsConversionMode;
   emit inputGarsConversionModeChanged();
+  emit pointToConvertChanged();
+  if (m_runConversion)
+    convertPoint();
```

Comment:
This seems confusing. The function `setInputGarsConversionMode` doesn't imply by reading the code that we should do this: `emit pointToConvertChanged();`.

Same for `setInputMgrsConversionMode` and `setInputUtmConversionMode`.